lib/playlist_id_migration_directions.md

```diff
@@ -0,0 +1,9 @@
+## Translate playlists from [this file](https://docs.google.com/spreadsheets/d/1gHHsqbiZNxeYeWApM7GhU4L8wiZuIJP3EvCd_hXk8xk/edit?usp=sharing) these ids were pulled to create the playlist id array. Some of the numbers in the google sheet aren't the most recent version of that playlist, so the spreadsheet and array don't exactly match. This file was last updated 11/14/17 - any playlists added to this file after that date are not included in this array. 
+
+1. `rake db:schema:load` clear the database and rebuild the sceme
+2. `pg_restore --verbose --clean --no-acl --no-owner -U h2oadmin -h h2o-alpha-94.coc4xoq8nz1r.us-east-1.rds.amazonaws.com -d h2o ~/h2o_prod-data.dump` port the data dump into the database
+2a. (for running locally: `pg_restore --verbose --clean --no-acl --no-owner -h localhost -d h2o_dev ~/h2o_prod-data.dump `)
```

Comment:
Same here

Reply:
Maybe don't include the address of the RDS instance...